packages/excalidraw/components/App.tsx

```diff
@@ -7878,16 +7882,25 @@ class App extends React.Component<AppProps, AppState> {
               suggestedBindings: getSuggestedBindingsForArrows(
                 selectedElements,
                 this.scene.getNonDeletedElementsMap(),
-              ),
+              )
             });
           }
-
+          // keyTest:(event)=>
+          window.addEventListener('keydown', (event) => {
```

Comment:
Instead of adding event listeners directly you should catch these in the `App::onKeyDown()`.

Reply:
thanks @mtolmacs